### PR TITLE
LPS-176897 Create input clear button

### PR DESCRIPTION
**Pull request by alvarolealdl**

### CODE REVIEW

**Comment by alvarolealdl** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/styles/components/_input.scss [hunk 1]
```diff
@@ -51,3 +51,13 @@ label {
...
+
+.search-filter {
+	padding-right: 45px;
+}
+
+.clear-button {
+	position: absolute;
+	right: 16px;
+	top: 42px;
+}
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/styles/components/_input.scss:63]:
```suggestion
.clear-button {
	position: absolute;
	right: 16px;
	top: 42px;
}

.search-filter {
	padding-right: 45px;
}
```

**Review by kevenleone — CHANGES_REQUESTED** [review]
LGTM 🔥 

@alvarolealdl just sort and fix the SF

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/ManagementToolbar/ManagementToolbarFilter.tsx [hunk 2]
```diff
@@ -126,12 +126,21 @@ const FilterBody = ({buttonRef, filterSchema, setPosition}: FilterBody) => {
...
 					value={filter}
 				/>
 
+				<ClayButtonWithIcon
+					aria-label="Clear"
+					className="clear-button"
+					displayType="unstyled"
+					onClick={() => setFilter('')}
+					symbol="times"
+					title="Clear"
```
Comment (samuelleolopes) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/ManagementToolbar/ManagementToolbarFilter.tsx:142]: title={i18n.translate('clear')}

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/styles/components/_input.scss (hunk 1 above)
Comment (alvarolealdl) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/styles/components/_input.scss]: Thanks @kevenleone , done.

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/ManagementToolbar/ManagementToolbarFilter.tsx (hunk 2 above)
Comment (alvarolealdl) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/ManagementToolbar/ManagementToolbarFilter.tsx]: Thanks @samuelleolopes  , done.

**Comment by alvarolealdl** [comment]
ci:test:sf

**Comment by alvarolealdl** [comment]
ci:test:sf

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**